### DISCUSSION TITLE
Dividing some_nature_of_contents and some_form_of_item

### DISCUSSION THREAD

**Discussion by dkreisstomkins** [uwlib-cams/uwlswd_vocabs_marc_006_008#25] [seeking feedback]
Should we split up some_nature_of_contents (formerly commonnat) into two vocabularies: continuing_nature_of_contents and books_nature_of_contents?

- Concepts within each have the same-ish definitions, but they don't overlap precisely in that some concepts are in one but not the other and visa versa.  
- They have different histories with different obsolete values.  
- They refer to different, but overlapping, marc position numbers.

**Comment by briesenberg07** [comment]
Is this basically the plan??
```
<> a skos:ConceptScheme;
   dct:title "Books: nature of contents" ;
   dct:alternative "values for MARC 008 character positions 24-27, where form of material is Books" ;
   ...

<> a skos:ConceptScheme;
   dct:title "Continuing resources: nature of contents" ;
   dct:alternative "values for MARC 008 character positions 25-27, where form of material is Continuing Resources" ;
   ...
```

**Reply by briesenberg07** [reply]
Per @dkreisstomkins this is basically the plan, although title and alternative values will vary.  
TO DO: Link from this discussion to the two new vocabs after PR is merged, tag Crystal and MARC2RDA people @briesenberg07
